bridge-fungible-token/src/events.sw

```diff
@@ -0,0 +1,36 @@
+library events;
+
+use std::{
+    contract_id::ContractId,
+    identity::Identity,
+    u256::U256,
+    vm::evm::evm_address::EvmAddress,
+};
+
+pub struct BurnEvent {
```

Comment:
A burn event might be a little redundant since we already have a `WithdrawalEvent` which implies a burn. Likewise, we should probably replace the `MintEvent` with a more verbose `DepositEvent` that also includes a `from` field.

Reply:
done.